store/copr/mpp.go

```diff
@@ -36,7 +36,7 @@ import (
 
 // MPPClient servers MPP requests.
 type MPPClient struct {
-	store *tikv.KVStore
```

Comment:
why not keep using anonymous field? I think it would look simpler.

Reply:
Because the anonymous field is not friendly for us to find which function in `KVStore` is used in the MPPClient, and we need to wrap the functions to make sure the error is converted to tidb err.